<launch>

    <node pkg="kf" type="ekfNode" name="extended_kalman_filter_node" output="screen">
        <rosparam command="load" file="$(find kf)/config/params.yaml"/>
    </node>

</launch>